### ISSUE TITLE
Check 3 "similar" questions in a row regarding interactions with multiple species

### ISSUE THREAD

**Issue by serenalomonico** [shcaba/FishPath-updates#21]
Questions 1318, 1457, and 1005 (sequential questions in the Management Measures section when the questionnaire is completed as Assessment, MM, DC) were noted to be similar in the Conil workshop. Need to check which options they trigger in FP before changing or editing wording.

![image](https://user-images.githubusercontent.com/16156813/39789877-6f6d8164-52e7-11e8-81c1-9aaeeadedb02.png)

![image](https://user-images.githubusercontent.com/16156813/39789883-726789dc-52e7-11e8-8f73-665c517ba80b.png)
![image](https://user-images.githubusercontent.com/16156813/39789886-761d3a90-52e7-11e8-9a7d-83150887bd67.png)

**Comment by bsnouffer** [comment]
Here is a xls showing all the caveats that each question currently trigger:
https://docs.google.com/spreadsheets/d/1wUa7OGvycW1SPEvOImQS4m5mevRHxP57uLt1G2gsJto/edit?usp=sharing

1318 triggers caveats in all 3 sections, 1457 and 1005 only trigger caveats in management measures. None are criteria for any options. 

Copying discussion prompt for 1318 here. I think it should be considered in the broader discussion of all three questions: 

> FishPath Team: There was large group discussion around the question's intent during the Conil workshop. Intent is to be proactive and answer “yes” conservatively. For example, if you are catching bocinegro, it’s also possible that you get voraz - in these terms, the fishery is multispecies.
> To resolve, check if this question invoke catch limits and other options? There are a lot of caveats on this question, not sure of the action (if any) needed other than a thorough check.

**Comment by bsnouffer** [comment]
Reviewed this when reviewing peer review comments as a similar suggestion was made. Decided to wrap 1457 into 1005, but leave 1318 as its own question. The caveats of 1457 and 1005 are very similar and the questions are also getting at very similar things, while 1318 is getting at a separate issue. We moved some of the advice given in the caveats from 1457 into the caveats of 1005. Here's the spreadsheet showing how the caveats of 1457 and 1005 are merged: https://docs.google.com/spreadsheets/d/1NCsoQcLC_d9HXOj-bfHu9nl-WN5DODGI2pRoCpaZ_K8/edit?usp=sharing


Updated in the Tool.
